backend/app/crud/crud_menu.py

```diff
@@ -26,15 +26,13 @@ async def get_all(self, db, name: str | None = None, status: int | None = None)
         menu = await db.execute(se)
         return menu.scalars().all()
 
-    async def get_role_menus(self, db, superuser: bool, menu_ids: list[int]) -> list[Menu]:
-        se = select(self.model).order_by(asc(self.model.sort))
-        where_list = [
-            self.model.menu_type.in_([0, 1]),
-            self.model.status == 1,
-        ]
-        if not superuser:
-            where_list.append(self.model.id.in_(menu_ids))
-        se = se.where(and_(*where_list))
+    async def get_role_menus(self, db, menu_ids: list[int]) -> list[Menu]:
+        se = (
+            select(self.model)
+            .where(self.model.id.in_(menu_ids))
+            .where(self.model.status == 1)
+            .order_by(asc(self.model.sort))
+        )
```

Comment:
Doesn't the superuser have direct access to all directories?

Reply:
Get the corresponding menu id from the user's role, the same is true for super users
```
            roles = request.user.roles
            menu_ids = []
            for role in roles:
                menu_ids.extend([menu.id for menu in role.menus])
```

Reply:
OK

Reply:
What is the point of supersuer? 
supersuer have all operational privileges by default, and so should menus. It is unrealistic to be forced to operate the database if the supersuer is modified by other supersuers